tests/datasets_tests/cub_tests/test_cub_label_dataset.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from chainer import testing
+from chainer.testing import attr
+
+from chainercv.datasets import cub_label_names
+from chainercv.datasets import CUBLabelDataset
+from chainercv.utils import assert_is_classification_dataset
+
+
+@testing.parameterize(
+    {'crop_bbox': True},
+    {'crop_bbox': False}
+)
+class TestCUBLabelDataset(unittest.TestCase):
+
+    def setUp(self):
+        self.dataset = CUBLabelDataset(crop_bbox=self.crop_bbox)
+
+    @attr.slow
+    def test_cub_label_dataset(self):
+        assert_is_classification_dataset(
+            self.dataset, len(cub_label_names), n_example=10)
+
```

Comment:
It is totally different thing, but what do you think about changing this dataset to return `bbox` instead of returning a cropped image.
This is more flexible because in some cases users want to crop an image by a padded bbox.

This can be done by replacing `crop_bbox` option to `return_bbox`. This style of interface is similar to `VOCDetectionDataset`, which also can return extra data.